src/Combinatorics/Graphs/functions.jl

```diff
@@ -698,6 +697,54 @@ julia> signed_incidence_matrix(g)
 """
 signed_incidence_matrix(g::Graph) = convert(Matrix{Int}, Polymake.graph.signed_incidence_matrix(pm_object(g)))
 
+# EdgeMap getters and setters for Edges
+function Base.getindex(M::Polymake.EdgeMap{TK, TV}, e::Edge) where {TK, TV}
+  return M[src(e), dst(e)]
+end
+
+function Base.setindex!(M::Polymake.EdgeMap{T, TV}, val, e::Edge) where {T, TV}
+  M[src(e), dst(e)] = val
+  return val
+end
+
+function Base.getindex(G::Graph{T}, i::Int, j::Int) where T
+  @req has_attribute(G, :edge_map) "Graph doesn't have an edge map set"
+  M = get_attribute(G, :edge_map)
+  return M[i, j]
+end
+Base.getindex(G::Graph{T}, index::NTuple{2, Int}) where T = G[index[1], index[2]]
+Base.getindex(G::Graph{T}, e::Edge) where T = G[src(e), dst(e)]
+
+function Base.setindex!(G::Graph{T}, val, i::Int, j::Int) where T
+  @req has_attribute(G, :edge_map) "Graph doesn't have an edge map set"
```

Comment:
```suggestion
  @req has_attribute(G, :edge_map) "Graph doesn't have an edge map set"
  @req has_edge(G, i, j) "invalid edge"
```
Polymake behaves slightly weird here as it automatically adds an edge if one tries to set a value for non-existing edge. So I think it would be better to block this here.
```